docs/contributing/Contributing-To-Ignite.md

```diff
@@ -22,6 +24,39 @@ node your-ignite-repo-clone/bin/ignite generate component OrderButton
 
 In addition, we have automated tests you can run with `yarn test`.
 
+### Making PRs
+
+We appreciate your contribution! To make it easier for us to review, please make sure to add a clear title and description that explains both what the change is, and why it's useful, if applicable.
+
+Maintainers may choose to rewrite this description for our changelog, or squash your commits when merging.
+
+## For maintainers
+
+### Merging PRs
+
+When merging PRs, we need maintainers to make sure that there's appropriately formatted commit message(s) for semantic-release to document changes for the release notes and pick the next-released version for Ignite. See [Releasing Ignite](./Releasing-Ignite.md) for more info on how to format commit messages.
+
+You have two choices when merging to Ignite:
+
+1. Merge with a merge commit, keeping the original commits. In this case, if the original commits are using the semantic-release conventions, those messages will be used for the changelog and release determinations.
```

Comment:
Weird these are all bulleted 1. but looks ok in your screenshot

Reply:
That's a markdown thing. You don't actually have to number them.

I usually do anyways, but the prettier markdown linter insisted they all be `1`.